lib/features/pokemon_list_page.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex/utils/constant.dart';
+
+class PokemonListPage extends StatelessWidget {
+  const PokemonListPage({super.key, required this.name});
+
+  final String name;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Pokedex'),
```

Comment:
kindly add a new `strings.dart` file inside your `utils` directory for this and make it as a string constant (i.e. `const String pokedexTitle  'Pokedex'`) 🙂